commands/spotify.js

```diff
@@ -0,0 +1,14 @@
+//$spotify
+//testing spotify web API
+const {MessageEmbed, Message} = require('discord.js');
+const fetch = require('node-fetch');
+
+exports.run = async (client,message,args) => {
+    let embed = new MessageEmbed()
+    try{
+        
+    } catch (err){
+        embed.setDescription('Error in spotify fxn: returning');
```

Comment:
We should either log, display to the user, or throw the error here instead of only printing a generic "Error in spotify fxn"